### ISSUE TITLE
resulution problem

### ISSUE THREAD

**Issue by onelz232** [MadsLeander/speedlimit#2]
when we installed this it worked for some people but for people with exempel 3440x1440 screen it was way off where it should be any fix for this

**Comment by MadsLeander** [comment]
Yeah, the script isn't made for other aspect ratios than 16:9. I don't have any plan to rewrite the CSS or lua code anytime soon, so you'll probably have to add support for ultra-wide yourself.
